packages/app-store/_appRegistry.ts

```diff
@@ -101,6 +101,7 @@ export async function getAppRegistryWithCredentials(userId: number, userAdminTea
     // Skip if app isn't installed
     /* This is now handled from the DB */
     // if (!app.installed) return apps;
+    app.createdAt = JSON.parse(JSON.stringify(dbapp.createdAt));
```

Comment:
```suggestion
    app.createdAt = app.createdAt.toISOString()
```

It's more clear what's happening this way.